### ISSUE TITLE
X2 not work mapper

### ISSUE THREAD

**Issue by NamedDroid** [gamesir123/app#69]
![Screenshot_20210821-003647_GameSir](https://user-images.githubusercontent.com/89266542/130258884-202e9d37-7dab-4206-9abe-a9dcdf6866f3.jpg)
Game start fail
G+Home key doesn't work

**Comment by Mcp20091** [comment]
Same issue here. I've tried various versions of the app with no luck. Tried turning off all accessibility options and running the mapper app from the site.

**Comment by RosiVF** [comment]
Same issue here. I just bought the gamepad and I cannot use it! Gamesir is not mapping, same as first screenshot, version 1.6-1.3, please fix it

**Comment by Mcp20091** [comment]
This is updated now and is working for me

**Comment by Leandro-Avila** [comment]
what did u do to work?

**Comment by manofsteel2017** [comment]
mine is working as well.
just update your app

**Comment by manofsteel2017** [comment]
latest version is 4.2.1

**Comment by Leandro-Avila** [comment]
where u find it?

**Comment by Mcp20091** [comment]
Correct. I installed 4.2 and when I opened it asked to update the app and then firmware for the controller. I am having issues with swapping A & B though for the Xbox app. It doesn't seem to stick.

**Comment by Leandro-Avila** [comment]
It worked, thanks!

**Comment by NamedDroid** [comment]
It worked too, but I can't make the button transparent. Is this possible?

**Comment by MuradPopatiya** [comment]
![Screenshot_2021-09-04-17-55-27-29_9e12f7ce48c7758dc08f890c09802501](https://user-images.githubusercontent.com/90091399/132090418-68759a5b-8efa-47e4-bc99-627b19732607.jpg)


"Mapping is not active"
Does anyone know how to fix this!!!

**Comment by endsha** [comment]
I am facing the same issue
I'm using Redmi Note 8 Pro with MIUI 12
It still says: "Mapping is not active"

**Comment by gothmandark** [comment]
Hello friends, I'm trying to run the GameSir app to play COD Mobile, but even though I press the start button, the game doesn't launch, no game launches. If I open COD Mobile directly, it recognizes the controller with the button configuration I had, but I can't map or change anything anymore. I can't see the transparent buttons. Does anyone know what it could be?
![f88fa5f3-1ff1-4a93-9065-48b35aabba88](https://github.com/gamesir123/app/assets/54748798/7d7da26b-58fe-44c7-a028-f83ac5dfcf07)
